.ci/docker/Dockerfile_dswx_ni

```diff
@@ -0,0 +1,66 @@
+Dockerfile to produce the production DSWx-NI PGE Docker image for OPERA
+# Authors: Scott Collins, Ray Bambery
+
+# Default SAS image path, must be provided by the docker build call via --build-arg
+ARG SAS_IMAGE
+FROM $SAS_IMAGE
+
+ARG PGE_SOURCE_DIR
+ARG PGE_DEST_DIR=/home/mamba
+ARG CONDA_ROOT=/opt/conda
+
+ENV PGE_DEST_DIR=$PGE_DEST_DIR
+ENV CONDA_ROOT=$CONDA_ROOT
+
+ARG BUILD_DATE_TIME
+ARG BUILD_VERSION
+
+# labels
+# the label-schema convention: http://label-schema.org/rc1/
+LABEL org.label-schema.build-date=${BUILD_DATE_TIME} \
+  org.label-schema.version=${BUILD_VERSION} \
+  org.label-schema.license="Copyright 2024,\
+ by the California Institute of Technology.\
+ ALL RIGHTS RESERVED.\
+ United States Government sponsorship acknowledged.\
+ Any commercial use must be negotiated with the Office of Technology Transfer\
+ at the California Institute of Technology.\
+ This software may be subject to U.S. export control laws and regulations.\
+ By accepting this document, the user agrees to comply with all applicable\
+ U.S. export laws and regulations. User has the responsibility to obtain\
+ export licenses, or other export authority as may be required, before\
+ exporting such information to foreign countries or providing access to\
+ foreign persons." \
+  org.label-schema.name="OPERA Product Generation Executable (PGE) Image" \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.vendor="California Institute of Technology" \
+  maintainer="California Institute of Technology"
+
+# Copy the OPERA PGE software into the container
+# the base container has a default user "mamba" with UID/GID 1000/1000
+COPY --chown=mamba:mamba ${PGE_SOURCE_DIR} ${PGE_DEST_DIR}
+
+# Switch to root for installing into Conda Env
+USER 0:0
+
+
+# Install dependencies into existing Conda Env
+# TODO: remove last chmod once permissions are set correctly on /home/mamba for delivered containers
+RUN set -ex \
```

Comment:
This is just a question.  The run commands in all the various Jenkins files are all different.  I know it's all based on what comes downstream to us, but will this always be the case, and if so is it worth trying to get things a little more standardized?

Reply:
Because the docker environments have not been particularly standardized in terms of directory structure and Python environments, I think were going to be stuck maintaining individualized run commands for each PGE.